Update magentatv.de.config.js

Sicilykill opened the pull request:
Added function to fetch cookie and csrfToken. 
The cookie and token are then set in the request.
This is an attempt to fix issue #2193 . 

Fixes #2193

freearhey commented:
Test failed:

<details>
<summary>Log</summary>

```sh
npm test -- magentatv.de

> test
> run-script-os magentatv.de


> test:default
> TZ=Pacific/Nauru npx jest --runInBand magentatv.de

 FAIL  sites/magentatv.de/magentatv.de.test.js
  ✓ can generate valid url (3 ms)
  ✓ can generate valid request method (1 ms)
  ✕ can generate valid request headers (2 ms)
  ✓ can generate valid request data (17 ms)
  ✕ can parse response (8 ms)
  ✓ can handle empty guide (1 ms)

  ● can generate valid request headers

    expect(received).toMatchObject(expected)

    Matcher error: received value must be a non-null object

    Received has type:  function
    Received has value: [Function headers]

      24 |
      25 | it('can generate valid request headers', () => {
    > 26 |   expect(request.headers).toMatchObject({
         |                           ^
      27 |     X_CSRFToken: 'e0a032d1c9df6c3fb8c8352399d32c40ddb17ccceb5142fe',
      28 |     'Content-Type': 'application/json',
      29 |     Cookie:

      at Object.<anonymous> (sites/magentatv.de/magentatv.de.test.js:26:27)

  ● can parse response

    expect(received).toMatchObject(expected)

    - Expected  - 2
    + Received  + 2

    @@ -5,10 +5,10 @@
            "Natur",
            "Tiere",
          ],
          "description": "Nachdem ein Mann von einem Sprengstoffpaket getötet wurde, das zu ihm nach Hause geschickt wurde, versucht das Team, den Absender zu fassen und sein neuestes tödliches Paket abzufangen. Maggie hat Mühe, ihrer jüngeren Schwester zu vertrauen.",
          "icon": "http://ngiss.t-online.de/sweetprogrammanager/media/gracenote/1/9/p19740197_e_h9_af.jpg",
    -     "start": "2022-03-09T00:00:00.000Z",
    -     "stop": "2022-03-09T00:45:00.000Z",
    +     "start": "2022-03-08T13:00:00.000Z",
    +     "stop": "2022-03-09T01:45:00.000Z",
          "title": "FBI: Special Crime Unit",
        },
      ]

      59 |   })
      60 |
    > 61 |   expect(result).toMatchObject([
         |                  ^
      62 |     {
      63 |       start: '2022-03-09T00:00:00.000Z',
      64 |       stop: '2022-03-09T00:45:00.000Z',

      at Object.<anonymous> (sites/magentatv.de/magentatv.de.test.js:61:18)

Test Suites: 1 failed, 1 total
Tests:       2 failed, 4 passed, 6 total
Snapshots:   0 total
Time:        3.989 s, estimated 4 s
Ran all test suites matching /magentatv.de/i.
```

</details>

Sicilykill commented:
Sorry about that. Forgot about the test.
Updated test.js to support the new method to fetch token and cookie.
I also updated the timezone of the session to point to UTC since it will impact how the programs start and end time are returned by the server. Something I had missed earlier
